script.js

```diff
@@ -1,63 +1,121 @@
 
-let fillCellsCounter, isXturn, TimeOutId;
+let fillCellsCounter, isXturn, TimeOutId, gameMode, isBotTurn, finished;
 const grid = document.getElementById('grid');
 const cells = Array.from(document.getElementsByClassName('cell'));
 const uses = cells.map((cell) => cell.firstElementChild);
 const svgStrikethrough = document.getElementById('strikethrough');
+const select = document.getElementsByName('gameMode')[0];
+const MODE = {
+    BOT_EASY: 'Easy bot',
+    BOT_HARD: 'Insane bot',
+    PVP_OFFLINE: 'Local PvP',
+    PVP_ONLINE: 'Online PvP',
+};
+
+// ========================= Setup =========================
+for (const mode in MODE) {
+    const option = document.createElement('option');
+    option.innerText = option.value = MODE[mode];
+    select.appendChild(option);
```

Comment:
I've also made the setting of game modes programmatical, so that we keep all the modes, their values and text in 1 place.